### PR TITLE
Release 0.0.2

### PR DESCRIPTION
**Pull request by pamella**
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ frontend TypeScript client, `django-ai-assistant-client`.
 ## 0.0.1 <small>June 25, 2024</small> {id="0.0.1"}
 
 - Initial release
+
+## 0.0.2 <small>June 28, 2024</small> {id="0.0.2"}
+
+- Add support for Django 4.2 LTS
+- Add support for Python 3.10 and 3.11
```
Comment (pamella) [inline CHANGELOG.md:18]: Should we mention the PRs or the changes made to the examples?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -102,8 +102,10 @@ poetry run mkdocs serve
 To release and publish a new version, follow these steps:
 
 1. Update the version in `pyproject.toml` and `frontend/package.json`.
-2. Update the changelog in `CHANGELOG.md`.
-3. Open a PR with the changes.
-4. Once the PR is merged, run the [Release GitHub Action](https://github.com/vintasoftware/django-ai-assistant/actions/workflows/release.yml) to create a draft release.
-5. Review the draft release, ensure the description has at least the associated changelog entry, and publish it.
-6. Once the review is publish, the [Publish GitHub Action](https://github.com/vintasoftware/django-ai-assistant/actions/workflows/publish.yml) will automatically run to publish the new version to [PyPI](https://pypi.org/project/django-ai-assistant) and [npm](https://www.npmjs.com/package/django-ai-assistant-client). Check the logs to ensure the publication was successful.
+2. In the project root, run `poetry run python manage.py generate_openapi_schema --output frontend/openapi_schema.json` to update the OpenAPI schema.
+3. In the frontend directory, run `pnpm run generate-client` to update the TypeScript client with the new OpenAPI schema.
```
Comment (pamella) [inline CONTRIBUTING.md:106]: Actual change.

File: CHANGELOG.md (hunk 1 above)
Comment (fjsj) [reply CHANGELOG.md:18]: Can you change the order? Newer versions on top.

Comment (fjsj) [reply CHANGELOG.md:18]: > Should we mention the PRs or the changes made to the examples?

No need to.

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comment.

**Review by fjsj — APPROVED** [review]

**Pull request merged**